src/components/AboutMission.tsx

```
@@ -0,0 +1,182 @@
+      <div className="left-side position-relative">
+        <img src={lftimg} alt="Factory Background" className="img-fluid w-100 h-100" />
+        <div id="personCard" className="card-overlay text-left text-white p-4">
+          <img
+            src={profiles[profileIndex].img}
+            className="rounded-circle mb-3 float-start imgshadow me-3"
+            width={120}
+            height={120}
+            alt="Person"
+          />
+          <p className="mb-2">{profiles[profileIndex].text}</p>
+          <h5 dangerouslySetInnerHTML={{ __html: profiles[profileIndex].name }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)